features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/Debouncer.kt

```diff
@@ -41,11 +46,31 @@ internal class Debouncer(
         lastTimeRecordWasPerformed = System.nanoTime()
     }
 
+    private fun runInTimeBalance(block: () -> Unit) {
+        if (timeBank.updateAndCheck(System.nanoTime())) {
+            val startTimeInNano = System.nanoTime()
+            block()
+            val endTimeInNano = System.nanoTime()
+            timeBank.consume(endTimeInNano - startTimeInNano)
+        } else {
+            // TODO: RUM-5188 report with telemetry about the frames skipped
+            internalLogger.log(
+                InternalLogger.Level.INFO,
+                InternalLogger.Target.USER,
```

Comment:
Is it useful for the user? What are they supposed to do if they see such log entry? Shouldn't it be `MAINTAINER` here?

Reply:
Here is a temporary handling of the skipped frames, and final solution will be report them with RUM session ended metrics. But you are right, in this case we'd better log to maintainer for now.